diegodofe/MoreGlean
# feature/populate-database

### What is this?
We've got some data now in our db!

Resolves #19 

### Steps to QA
1. **Navigate** to the /create page.
2. **Ensure** that docs are being added to Firestore when you click the buttons.
